plugins/nodes/src/js/pages/nodes/NodeDetailPage.js

```diff
@@ -83,7 +83,7 @@ class NodeDetailPage extends mixin(TabsMixin, StoreMixin) {
     this.updateCurrentTab(nextProps);
   }
 
-  componentWillUpdate() {
+  UNSAFE_componentWillUpdate() {
     super.componentWillUpdate(...arguments);
```

Comment:
shouldn't we change those calls as well?

Reply:
We should, this PR is still work in progress.